lib/std/crypto/modes.zig

```diff
@@ -0,0 +1,60 @@
+// SPDX-License-Identifier: MIT
+// Copyright (c) 2015-2020 Zig Contributors
+// This file is part of [zig](https://ziglang.org/), which is MIT licensed.
+// The MIT license requires this copyright notice to be included in all copies
+// and substantial portions of the software.
+// Based on Go stdlib implementation
+
```

Comment:
This should get a doccomment of what "mode" is.

Reply:
I'm going to add a comment, thanks!

Note that everything in `crypto/core` is not supposed to be directly used by applications, unless they really need to implement their own constructions. This includes modes and the core AES function.

These are just building blocks for higher-level constructions such as AEADs.

Reply:
Added, thanks!